service/controller/aws/v20/resource_set.go

```diff
@@ -23,13 +24,11 @@ import (
 	"github.com/giantswarm/cluster-operator/pkg/label"
 	chartconfigservice "github.com/giantswarm/cluster-operator/pkg/v20/chartconfig"
 	configmapservice "github.com/giantswarm/cluster-operator/pkg/v20/configmap"
+	"github.com/giantswarm/cluster-operator/pkg/v20/resource/app"
 	"github.com/giantswarm/cluster-operator/pkg/v20/resource/certconfig"
-	"github.com/giantswarm/cluster-operator/pkg/v20/resource/chartoperator"
 	"github.com/giantswarm/cluster-operator/pkg/v20/resource/clusterconfigmap"
 	"github.com/giantswarm/cluster-operator/pkg/v20/resource/encryptionkey"
 	"github.com/giantswarm/cluster-operator/pkg/v20/resource/kubeconfig"
-	"github.com/giantswarm/cluster-operator/pkg/v20/resource/namespace"
-	"github.com/giantswarm/cluster-operator/pkg/v20/resource/tiller"
```

Comment:
Why do we also delete these resources when the PR is only about `chart-operator`?

Reply:
As Ross mentioned in this comment, https://github.com/giantswarm/cluster-operator/pull/644#pullrequestreview-282612439 we need a proper test on when `chart-operator` resource got removed. We already move these parts to app-operator, would like to see that it's going well w/o chart-operator and other resources. (namespace, tiller)